Ginger/GingerCoreNET/RunLib/CLILib/CLIScriptFile.cs

```diff
@@ -30,7 +29,7 @@ public class CLIScriptFile : ICLI
 
         string mScriptFile;
 
-        public string Identifier
+        public string Verb
         {
             get
             {
```

Comment:
should come back from ScriptOptions.Verb

Reply:
fixed